packages/test-utils/src/testInterface.ts

```diff
@@ -0,0 +1,10 @@
+interface HavingTestInterface { testInterfaceType?: any }
+
+/**
+ * Gets the Testing Interface
+ * @param obj The object to get the testing interface from
+ * @returns The testing interface
+ */
+export function getTI<T extends HavingTestInterface> (obj: T): NonNullable<T['testInterfaceType']> {
```

Comment:
Rename to `getTestInterface` as TI is not a well-known acronym (and therefore not known to those who read our tests)

Reply:
done